README.md

```diff
@@ -14,7 +14,9 @@ The Whisper model version can be adjusted in the `config.yml` file by editing th
 |`large-v2`|1550 M|
 |`large-v3`|1550 M|
 
-We recommend version `large-v2` as it performs better in most cases than `large-v3`.
+We recommend version `large-v2` as it performs better than `large-v3` in our benchmarks.
+
+You can also specify an S3 URI bucket if you have your own custom model available in an S3 bucket.
```

Comment:
Either a S3 URI or a S3 bucket - but I think in this case one should specify a URI